cadctl/cmd/cluster-missing/cluster-missing.go

```diff
@@ -55,17 +56,34 @@ var ClusterMissingCmd = &cobra.Command{
 			return fmt.Errorf("could not load CAD_PD envvar")
 		}
 
-		_, err = pagerduty.NewWithToken(CAD_PD)
+		pd, err := pagerduty.NewWithToken(CAD_PD)
 		if err != nil {
 			return fmt.Errorf("could not start pagerdutyClient: %w", err)
 		}
 
+		res, err := chgm.AreAllInstancesRunning(externalClusterID)
+		if err != nil {
+			return fmt.Errorf("could not verify all instances were running: %w", err)
+		}
+		fmt.Printf("yay! the result is %#v\n", res)
+
+		if res {
+			fmt.Println("The node shutdown was not the customer. Nothing to do.")
+			return nil
+		}
+
+		err = pd.AddNote(incidentID, "TODO_NOTE_CONTENT")
```

Comment:
some format I have in another script (can be modified)
```
PD_CHGM_NOTE='User-initiated shutdown of nodes in the cluster. Servicelog posted. Silencing alert.'
```
your flow doesn't only do the PD note, it does more.
I guess you can pull some of the logic out of my PR (how I did the client nesting and such) so you can do tyour whole flow in one func

Reply:
In our Ticket @rafael-azevedo  specified that we should attach the full service log on the alert. Do you know how I get that service log? I see that it's implemented in the ocm package, but we have no function yet to retrieve it :thinking:

Reply:
he meant to use the ocm command but:
- instead of pulling from GH, have it saved locally
- remove the code that pulls from GH (as now it will exist inside the codebase

Reply:
Well, I can tackle this in my PR, too.

Reply:
I tackled this in a separate PR, because i think it's cleaner: https://github.com/openshift/configuration-anomaly-detection/pull/39